td-agent/Rakefile

```diff
@@ -544,6 +615,74 @@ class BuildTask
     end
   end
 
+  def openssldir
+    File.join(staging_etcdir, "openssl")
+  end
+
+  def create_certs
+    keychains = [
+      "/System/Library/Keychains/SystemRootCertificates.keychain"
+    ]
+
+    cert_list, error, status = Open3.capture3("security find-certificate -a -p #{keychains.join(' ')}")
+    unless status.success?
+      puts(error: error)
+      exit 1
```

Comment:
We use `fail` method (alias of `raise`) for an error case.

Reply:
Oh! I see.